packages/Algernon.package/ALGItemProvider.class/instance/handleSystemChangeEvent..st

```diff
@@ -0,0 +1,4 @@
+updating
+handleSystemChangeEvent: anEvent
+	self class initializeItems.
+	self cache clear
```

Comment:
I'm not really sure whether it's really such a great idea to handle it like this. On the other hand, everything still runs smoothly (again kinda proving the uselessness of this weird cache??). 
Anyway, I feel like the appropriate solution would be to make the ClassType dynamic. But then again ... This would ultimately disable the cache for all classes, whereas I guess we can expect the manual cache clear because of the creation/removal of a class not to happen too often. It's a difficult consideration between the harm done by not caching classes vs trashing the whole cache (kinda) regularly😕
In the end, I feel like it pretty much does not matter because we happen to have very few benefits from the cache anyway...
What do you think?🤔

Reply:
Yeah, it feels more like removing a symptom by clearing the whole cache than really dealing with the root cause (locating which parts of the cache need to be exactly cleared when new classes appear). But as far as my understanding of the fuzzy matching goes, we lookup the cache for every new character that is typed in the search bar in order to achieve sth like substring-matching. And at least for me that makes it kinda unpredictable which parts of the cache really need to be cleared.

As you noted adding/removing classes won't happen that much, so that was our consideration for just clearing the whole cache on those occasions.